k8s/backend-simple.yaml

```
@@ -0,0 +1,371 @@
+        - -c
+        - |
+          # Set environment
+          export npm_config_cache=/tmp/.npm
+          export HOME=/tmp
+          export NODE_ENV=development
+          export PORT=3002
+          
+          # Create a minimal backend server
+          mkdir -p /tmp/app
+          cd /tmp/app
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)